packages/components/src/DateTimePickers/MonthPicker/MonthPicker.scss

```diff
@@ -0,0 +1,29 @@
+
+.container {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
```

Comment:
Remove that please.
You manage the wrap by splitting the months in rows. No need that.

Reply:
&#10003;